app/components/howItWorksModal.tsx

```diff
@@ -0,0 +1,108 @@
+          <ol className={styles.list}>
+            <li>
+              <strong>Pick Your Colors:</strong> Click on the two color pickers
+              to select your primary and secondary colors. These could be your
+              favorite colors, your company colors, or any combination you love!
+            </li>
+            <li>
+              <strong>See Your Matches:</strong> We&apos;ll instantly show you
+              sports teams that have similar color schemes, sorted by how
+              closely they match your choices.
+            </li>
+            <li>
+              <strong>Adjust Results:</strong> Use the slider at the bottom to
+              see more or fewer teams (1-20 teams).
+            </li>
+            <li>
+              <strong>Explore Teams:</strong> Click on any team to learn more
+              about them!
+            </li>
+          </ol>
+        </section>
+
+        <section className={styles.section}>
+          <h3 className={styles.sectionTitle}>How Color Matching Works</h3>
+          <p className={styles.paragraph}>
+            We use the <strong>Delta E algorithm</strong>, a scientific method
+            that measures how the human eye perceives color differences. This
+            means we don&apos;t just compare RGB values - we find teams whose
+            colors
+            <em> look</em> similar to your choices, just like your eyes would
+            see them!
+          </p>
+          <p className={styles.paragraph}>The algorithm considers:</p>
+          <ul className={styles.list}>
+            <li>Hue (the actual color)</li>
+            <li>Saturation (how vivid the color is)</li>
+            <li>Brightness (how light or dark it is)</li>
+          </ul>
+        </section>
+
+        <section className={styles.section}>
+          <h3 className={styles.sectionTitle}>Sports & Leagues Included</h3>
+          <p className={styles.paragraph}>
+            Our database includes teams from major sports leagues around the
+            world:
+          </p>
+          <ul className={styles.list}>
+            <li>🏈 NFL (American Football)</li>
+            <li>🏀 NBA (Basketball)</li>
+            <li>⚾ MLB (Baseball)</li>
+            <li>🏒 NHL (Hockey)</li>
+            <li>⚽ Soccer/Football leagues worldwide</li>
+            <li>🏏 Cricket teams</li>
+            <li>🏉 Rugby teams</li>
+            <li>And many more!</li>
+          </ul>
+        </section>
+
+        <section className={styles.section}>
+          <h3 className={styles.sectionTitle}>Pro Tips</h3>
+          <ul className={styles.list}>
+            <li>
+              <strong>Try complementary colors</strong> for interesting matches
+              - teams often use colors that pop together!
+            </li>
+            <li>
+              <strong>Use the randomize button</strong> (🎲) to discover
+              unexpected color combinations and teams.
+            </li>
```

Comment:
There are four actions (swap, random existing colors, tottaly random colors, and reset, in the `ColorActions` component). Explain them all like you did with the randomize, but instead of a random emoji add the small icon they are using (`FaExchangeAlt`, `FaBullseye`, `FaRegQuestionCircle`, `FaRegTimesCircle`)